<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="2" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Application is set up successfully]]>
    </line>
    <line>
      <![CDATA[Browser Session End]]>
    </line>
    <line>
      <![CDATA[Application is set up successfully]]>
    </line>
    <line>
      <![CDATA[Browser Session End]]>
    </line>
  </reporter-output>
  <suite name="TestNG Listeners on Failure" duration-ms="46592" started-at="2019-10-25T00:28:07 PDT" finished-at="2019-10-25T00:28:53 PDT">
    <groups>
    </groups>
    <test name="Login Test" duration-ms="46592" started-at="2019-10-25T00:28:07 PDT" finished-at="2019-10-25T00:28:53 PDT">
      <class name="com.test.test07.listeners.onfailure.screenshot.xml.ScreenshotFromXmlTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.test.test07.listeners.onfailure.screenshot.xml.ScreenshotFromXmlTest@3dfc5fb8]" name="setUp" is-config="true" duration-ms="12738" started-at="2019-10-25T00:28:07 PDT" finished-at="2019-10-25T00:28:19 PDT">
          <reporter-output>
            <line>
              <![CDATA[Application is set up successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="doLogin()[pri:1, instance:com.test.test07.listeners.onfailure.screenshot.xml.ScreenshotFromXmlTest@3dfc5fb8]" name="doLogin" duration-ms="20091" started-at="2019-10-25T00:28:19 PDT" finished-at="2019-10-25T00:28:40 PDT">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[no such element: Unable to locate element: {"method":"css selector","selector":"#identifierIdError"}
  (Session info: chrome=77.0.3865.90)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'DESKTOP-7F3VJB9', ip: '192.168.88.216', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_66'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 77.0.3865.90, chrome: {chromedriverVersion: 77.0.3865.40 (f484704e052e0..., userDataDir: C:\Users\yulia\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:8675}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: 6efba3616e7f4b5f3ecefb45c9ae09fd
*** Element info: {Using=id, value=identifierIdError}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"css selector","selector":"#identifierIdError"}
  (Session info: chrome=77.0.3865.90)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'DESKTOP-7F3VJB9', ip: '192.168.88.216', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_66'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 77.0.3865.90, chrome: {chromedriverVersion: 77.0.3865.40 (f484704e052e0..., userDataDir: C:\Users\yulia\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:8675}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: 6efba3616e7f4b5f3ecefb45c9ae09fd
*** Element info: {Using=id, value=identifierIdError}
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.findElementById(RemoteWebDriver.java:372)
at org.openqa.selenium.By$ById.findElement(By.java:188)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at com.test.test07.listeners.onfailure.screenshot.xml.ScreenshotFromXmlTest.doLogin(ScreenshotFromXmlTest.java:26)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.test.test07.listeners.onfailure.screenshot.xml.ScreenshotFromXmlTest@3dfc5fb8]" name="tearDown" is-config="true" duration-ms="693" started-at="2019-10-25T00:28:40 PDT" finished-at="2019-10-25T00:28:41 PDT">
          <reporter-output>
            <line>
              <![CDATA[Browser Session End]]>
            </line>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.test.test07.listeners.onfailure.screenshot.xml.ScreenshotFromXmlTest@3dfc5fb8]" name="setUp" is-config="true" duration-ms="11455" started-at="2019-10-25T00:28:41 PDT" finished-at="2019-10-25T00:28:52 PDT">
          <reporter-output>
            <line>
              <![CDATA[Application is set up successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="composeEmail()[pri:2, instance:com.test.test07.listeners.onfailure.screenshot.xml.ScreenshotFromXmlTest@3dfc5fb8]" name="composeEmail" duration-ms="4" started-at="2019-10-25T00:28:52 PDT" finished-at="2019-10-25T00:28:52 PDT">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Error in composing email]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Error in composing email
at org.testng.Assert.fail(Assert.java:97)
at com.test.test07.listeners.onfailure.screenshot.xml.ScreenshotFromXmlTest.composeEmail(ScreenshotFromXmlTest.java:37)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- composeEmail -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.test.test07.listeners.onfailure.screenshot.xml.ScreenshotFromXmlTest@3dfc5fb8]" name="tearDown" is-config="true" duration-ms="668" started-at="2019-10-25T00:28:53 PDT" finished-at="2019-10-25T00:28:53 PDT">
          <reporter-output>
            <line>
              <![CDATA[Browser Session End]]>
            </line>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.test.test07.listeners.onfailure.screenshot.xml.ScreenshotFromXmlTest -->
    </test> <!-- Login Test -->
  </suite> <!-- TestNG Listeners on Failure -->
</testng-results>
